EIPS/eip-draft_sft.md

```diff
@@ -0,0 +1,312 @@
+---
+eip: <to be assigned>
```

Comment:
```suggestion
eip: 4635
```
Note: Please also change the filename to `EIPS/eip-4635.md`.

Reply:
I've updated the file with 4635. Thanks